ZUGFeRD-Test/Application.cs

```diff
@@ -28,7 +28,7 @@ internal void Run()
         {
             _loadZUGFeRD2Invoice();
 
-            InvoiceDescriptor desc = InvoiceDescriptor.Load(@"zugferd.xml");
+            InvoiceDescriptor desc = InvoiceDescriptor.Load(@"..\..\..\demodata\zugferd10\ZUGFeRD_1p0_BASIC_Einfach.xml");
```

Comment:
Let's just remove this line. 
ZUGFeRD_1p0_BASIC_Einfach.xml is used in another function of the sample application.